concord_ml/bcd/computations.py

```diff
@@ -1,27 +1,64 @@
 """Local implementation for Bayesian Changepoint Detection"""
-
 import numpy as np
+from concord.computation import (
+    Computation,
+    Metadata
+)
 
+# import logging
+# logging.basicConfig()
+# logger = logging.getLogger(__name__)
+# logger.setLevel(logging.DEBUG)
 
-class BayesianChangepointDetection(object):
-    """ Bayesian Changepoint Detection implementation
 
+class BayesianChangepointDetection(Computation):
+    """ Bayesian Changepoint Detection implementation
     Based on algorithm from http://arxiv.org/abs/0710.3742.
+    """
+    def __init__(self, hazard, distribution, istream=None, ostream=None):
```

Comment:
`istream` really shouldn't ever be `None`. That'd be really silly.

I also feel like `ostream` shouldn't be `None` either, but who knows?

Also, if we're just going to pass in `[] if ostrema is None else ostream`, why don't we just set the default an empty tuple `()`?